USAGE_DATA.md

```diff
@@ -13,6 +13,12 @@ When telemetry events are enabled, the following information is emitted when the
     * The free, total, and max VM memory
  * Version information:
     * The server version number
+ * Text Document Information
+   * When a document is opened :
+      * The file extension (eg. `xml`, `xsd`, `dtd`)
+      * The associated grammar types (eg. `none`, `doctype`, `xml-model`, `xsi:schemaLocation`, `xsi:noNamespaceSchemaLocation`)
+      * The system/public identifier used for the grammar, if any
```

Comment:
After though i wonder if its a relevant information? Its more a data user for me.  I will remove it.

Reply:
Sounds fine. We can remove the system/public identifier for now. Could this not be used to determine if the schema is a local file or remote ?

Reply:
Done. I've removed the identifier uri.